### ISSUE TITLE
Export to .ged from DB

### ISSUE THREAD

**Issue by AmirJavaidSwe** [agjmills/laravel-gedcom#1]
Is there any way to export the imported data to a human-readable format, or to .ged

**Comment by agjmills** [comment]
@AmirJavaidSwe yep - that should be possible, it would just need to do what `src/Utils/GedcomParser.php` does but backwards
